src/main/java/it/gov/pagopa/cgn/portal/service/BucketService.java

```diff
@@ -73,9 +123,10 @@ public void setRunningBucketLoad(Long discountId) {
         }
     }
 
-    @Transactional(Transactional.TxType.REQUIRES_NEW)
+    @Transactional(Transactional.TxType.REQUIRED)
```

Comment:
we must test this type of transaction in order to avoid regressions

Reply:
Tests are correctly working, anyway this is refactor typo remained there after some testing. I will check if we need to rollback outside context in case of failure or if we just need a nested transaction.

Reply:
I checked transaction and set 3 of them to requires_new because they are done in asynchronous code. https://github.com/pagopa/cgn-onboarding-portal-backend/pull/107/commits/9e9779ea784ca535665776493e2eb2b14a7bb3b3